Sources/TestHelpers/UnitTestService.swift

```diff
@@ -0,0 +1,16 @@
+
+public class UnitTestService <U>: Serveable {
```

Comment:
I don't think this file should reside inside the pod. I would include it in the test target of the example project.